doc/page-cmakefunctions.h

```diff
@@ -0,0 +1,173 @@
+/**
+
+\page CMake-Functions FaceLift CMake API Reference
+
+
+\section public-functions Functions for FaceLift users
+
+Those functions are for convinience and are meant to be used by FaceLift users.
```

Comment:
convinience

Reply:
changing the word .

Idea was to convey that its a convenient way for end user to use following functions